app.rb

```diff
@@ -0,0 +1,193 @@
+require 'date'
+require './student'
+require './teacher'
+require './rental'
+require './book'
+require './classroom'
+
+class App
+  attr_reader :books, :people, :rentals
+
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def list_all_books
+    return puts '0 Books currently available' if @books.empty?
+
+    @books.each_with_index do |book, index|
+      puts "#{index + 1}. Title: #{book.title} by Author: #{book.author}"
+    end
+  end
+
+  def list_all_people
+    return puts 'No people Registered' if @people.empty?
+
+    @people.each_with_index do |person, index|
+      puts "#{index + 1}. [#{person.class}], id: #{person.id}, Name: #{person.name}, Age: #{person.age}"
```

Comment:
When listing all people, the name for a teacher doesn't appear as it should: 
![image](https://github.com/ProgramKingAbel/oop-school-lib/assets/93520178/d19419c2-89e0-46ee-8220-1fbbbf5c45e0)
Kindly fix this issue.